enterprise/admin/__init__.py

```diff
@@ -48,17 +51,58 @@ class EnterpriseCustomerIdentityProviderInline(admin.StackedInline):
     form = EnterpriseCustomerIdentityProviderAdminForm
 
 
+class EnterpriseCustomerEntitlementInline(admin.StackedInline):
+    """
+    Django admin model for EnterpriseCustomerEntitlement.
+    The admin interface has the ability to edit models on the same page as a parent model. These are called inlines.
+    https://docs.djangoproject.com/en/1.8/ref/contrib/admin/#django.contrib.admin.StackedInline
+    """
+
+    model = EnterpriseCustomerEntitlement
+    extra = 0
```

Comment:
The number of empty forms that is displayed is controlled by the extra parameter.